RecoLocalCalo/EcalRecAlgos/interface/EcalUncalibRecHitTimingCCAlgo.h

```diff
@@ -16,31 +16,28 @@
 #include "RecoLocalCalo/EcalRecAlgos/interface/EigenMatrixTypes.h"
 
 class EcalUncalibRecHitTimingCCAlgo {
-  float startTime_;
-  float stopTime_;
-  float targetTimePrecision_;
-
-  static constexpr int TIME_WHEN_NOT_CONVERGING = 100;
-  static constexpr int MAX_NUM_OF_ITERATIONS = 30;
-  static constexpr int MIN_NUM_OF_ITERATIONS = 2;
-  static constexpr float GLOBAL_TIME_SHIFT = 100;
-
 public:
-  EcalUncalibRecHitTimingCCAlgo(const float startTime = -5,
-                                const float stopTime = 5,
-                                const float targetTimePrecision = 0.001);
-  ~EcalUncalibRecHitTimingCCAlgo(){};
+  EcalUncalibRecHitTimingCCAlgo(const float startTime, const float stopTime, const float targetTimePrecision);
   double computeTimeCC(const EcalDataFrame& dataFrame,
                        const std::vector<double>& amplitudes,
                        const EcalPedestals::Item* aped,
                        const EcalMGPAGainRatio* aGain,
                        const FullSampleVector& fullpulse,
                        EcalUncalibratedRecHit& uncalibRecHit,
-                       float& errOnTime);
+                       float errOnTime) const;
```

Comment:
```suggestion
                       float& errOnTime) const;
```
I noticed only now thanks to the static analyzer https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-74f75c/13543/llvm-analysis/report-9ba007.html#EndPath.
Please